styles/pov.h

```diff
@@ -4,14 +4,48 @@
 // TODO(hubbe): Make it work with FastLED
 #if defined(ENABLE_WS2811)
 
-// Usage: &style_pov or StylePtr<StylePtr>
-// return value: suitable for preset array
-
-// This style draws images in the air if you swing the saber
-// carefully back and forth. Unfortunately you can't really see
-// the result without a slow exposure camera. It's possible that
-// dotstar APA102 pixels would be fast enough to do the effect
-// full justice, but I haven't tried that.
+/* 
+This style draws images in the air if you swing the saber
+carefully back and forth. Unfortunately you can't really see
+the result without a slow exposure camera. It's possible that
+dotstar APA102 pixels would be fast enough to do the effect
+full justice, but I haven't tried that.
+
+Usage: &style_pov
+or: StylePtr<StylePOV<>>()
+or: StylePtr<StylePOV<MIN_DEGREES, MAX_DEGREES, REPEAT, MIRROR>>()
+MIN_DEGREES: integer (default -45)
+MAX_DEGREES: integer (default 45)
+REPEAT: bool (default false)
+MIRROR: bool (default true)
+return value: COLOR
+Since MIN_DEGREES, MAX_DEGREES, REPEAT and MIRROR all have defaults, those arguments can be omitted.
+StylePOV<> draws one instance of the image in the top portion of the swing "window".
+
+Usage:StylePtr<ContinuousPOV<>>()
+or: StylePtr<ContinuousPOV<DEGREES>>()
+DEGREES: integer (default 90)
+return value: COLOR
+Omitting the DEGREES argument uses the default value of 90.
+ContinuousPOV<> draws the image multiple times continuously through 360 degrees of the swing.
```

Comment:
People might be confused about the difference between ContinuousPOV and StylePOV<> with REPEAT=true, so we should probably explain the difference, something like:

StylePOV tries to always display the same line when the saber is at a particular angle.
ContinuousPOV simply moves from line to line, and the speed is controlled by the swing speed.
If you move the saber left and then right, StylePOV will go *back*, ContinuousPOV will continue forward.